### PR TITLE
Update cli init username

### PR DESCRIPTION
**Pull request by themorey**
Modified cli initialize function from hardcoded username to variable.

@bwatrous I also modified `main.tf` to pull the script from my repo but that's not needed in a merge.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -107,7 +107,7 @@ resource "azurerm_virtual_machine_extension" "fetch_cyclecloud_install_script" {
 
   settings = <<SETTINGS
     {
-        "commandToExecute": "curl -k -L -o /tmp/install_cyclecloud.py \"https://raw.githubusercontent.com/bwatrous/cyclecloud-terraform/master/scripts/cyclecloud_install.py\""
+        "commandToExecute": "curl -k -L -o /tmp/install_cyclecloud.py \"https://raw.githubusercontent.com/themorey/cyclecloud-terraform/master/scripts/cyclecloud_install.py\""
```
Comment (bwatrous) [inline main.tf]: Can we revert this change for master?

Comment (themorey) [reply main.tf]: i updated it in my repo
